plasmapy/particles/tests/test_decorators.py

```diff
@@ -593,6 +601,22 @@ def test_particle_list_all_in_category(self, case):
         case.function(case.particles_in_category)
 
 
+def test_custom_particle_for_parameter_named_ion():
+    """
+    Test that a positively charged CustomParticle is treated as a valid
+    ion when the parameter is named ``ion``.
+    """
+    custom_ion = CustomParticle(mass=2e-27 * u.kg, charge=3e-19 * u.C)
+    result = get_ion(custom_ion)
+    assert result == custom_ion
+
+
+def test_creating_mean_particle_for_parameter_named_ion():
+    Z = 1.3
+    ion = get_ion(ion="He", Z=Z)
+    assert u.isclose(ion.charge, Z * const.e.si)
+
```

Comment:
Do you need an additional test here enforcing that this statement is true? 

```
|particle_input| no longer enforces that |parameters| named
``ionic_level`` are ions or neutral atoms. For equivalent behavior,
name the parameter ``ion`` instead.
```

Something that fails to raise an error for `ionic_level` but does for `ion`?

Reply:
Given the obscurity of `ionic_level`, I'd say we can safely skip this test.